README.md

```diff
@@ -320,6 +421,8 @@ dataset (_dataset_ parameter) table with the name of input table as suffix.
 | `inspectTemplateName`            | DLP inspect template name                                                                                                                                                                                                                                          | INSPECT/DEID        | 
 | `deidentifyTemplateName`         | DLP de-identify template name                                                                                                                                                                                                                                      | All                 |
 | `DLPMethod`                      | Type of DLP operation to perform - INSPECT/DEID/REID                                                                                                                                                                                                               | All                 |
+| `processExistingFiles`           | If set to false then the existing files in GCS bucket before pipeline is triggered will not be processed (default value is true)                                                                                                                                                                                                               | INSPECT/DEID                 |
+| `gcsNotificationTopic`           | Name of Pub/Sub topic through which new files ingested in GCS bucket will be notified (should be provided in the form projects/$PROJECT_ID>/topics/$GCS_TOPIC_NAME)                                                                                                                                                                                                               | INSPECT/DEID                 |
```

Comment:
How about:
Pub/Sub topic to read notifications of new files added to the GCS bucket in **_filePattern_**.

Reply:
Will fix